flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
this is disappointing.

Reply:
arm coming in the very shortterm. I just added arm builds for the server version and client automation will come next. I'm waiting somewhat until Github implements arm runners for open source projects. Currently I use Runson runners that I need to pay for.

Reply:
flathub has arm builders. The suggestion to build from source is based on that too.

Reply:
I don't think I can do that with what Tauri currently supports. They specifically compile only to deb, rpm, and appimage files. Additionally, it's quite a complex toolchain that would need to run where rust would compile down to webpack using trunk and then be bundled by tauri. 

If not having arm is a blocker I can fast track that and do it right away but I don't think I would be able to compile from source.

Reply:
I was able to pretty quickly build out a job that builds arm clients. That has now been implemented and I added the arm client on as well. We build on both.